pkg/security/apiserver/admission/sccadmission/admission_test.go

```diff
@@ -823,29 +825,39 @@ func TestAdmitWithPrioritizedSCC(t *testing.T) {
 	uidSix := int64(6)
 	matchingPriorityAndScoreSCCOne := laxSCC()
 	matchingPriorityAndScoreSCCOne.Name = "matchingPriorityAndScoreSCCOne"
-	matchingPriorityAndScoreSCCOne.RunAsUser = securityapi.RunAsUserStrategyOptions{
-		Type: securityapi.RunAsUserStrategyMustRunAs,
+	matchingPriorityAndScoreSCCOne.RunAsUser = securityv1.RunAsUserStrategyOptions{
+		Type: securityv1.RunAsUserStrategyMustRunAs,
 		UID:  &uidSix,
 	}
 	matchingPriorityAndScorePriority := int32(1)
 	matchingPriorityAndScoreSCCOne.Priority = &matchingPriorityAndScorePriority
 
 	matchingPriorityAndScoreSCCTwo := laxSCC()
 	matchingPriorityAndScoreSCCTwo.Name = "matchingPriorityAndScoreSCCTwo"
-	matchingPriorityAndScoreSCCTwo.RunAsUser = securityapi.RunAsUserStrategyOptions{
-		Type: securityapi.RunAsUserStrategyMustRunAs,
+	matchingPriorityAndScoreSCCTwo.RunAsUser = securityv1.RunAsUserStrategyOptions{
+		Type: securityv1.RunAsUserStrategyMustRunAs,
 		UID:  &uidSix,
 	}
 	matchingPriorityAndScoreSCCTwo.Priority = &matchingPriorityAndScorePriority
 
 	// we will expect these to sort as:
 	expectedSort := []string{"restrictive", "matchingPrioritySCCOne", "matchingPrioritySCCTwo",
 		"matchingPriorityAndScoreSCCOne", "matchingPriorityAndScoreSCCTwo"}
-	sccsToSort := []*securityapi.SecurityContextConstraints{matchingPriorityAndScoreSCCTwo, matchingPriorityAndScoreSCCOne,
+	sccsToSort := []*securityv1.SecurityContextConstraints{matchingPriorityAndScoreSCCTwo, matchingPriorityAndScoreSCCOne,
 		matchingPrioritySCCTwo, matchingPrioritySCCOne, restricted}
-	sort.Sort(oscc.ByPriority(sccsToSort))
 
-	for i, scc := range sccsToSort {
+	internalSccsToSort := []*securityapi.SecurityContextConstraints{}
+	for _, externalConstraint := range sccsToSort {
```

Comment:
Why not just have a test-only external sorter?

Reply:
Thanks, converted the existing one to externals